background.js

```diff
@@ -72,6 +72,7 @@ function downloadAgent() {
                         ]
                     };
                     chrome.notifications.create( Math.round((new Date()).getTime() / 1000).toString(), notificationOptions);
+                    console.log(setTimeout(() => chrome.runtime.reload(), 1000));
```

Comment:
@149segolte Why do we need this console.log and setTimeout?

Reply:
It is calling reload every 1-second won't that have any performance implications

Reply:
The reload is only triggered when the download gets successfully initiated therefore only runs when the user downloads something.

As for the timeout, the reload should trigger after notification is issued otherwise the notification will not be displayed therefore the timeout is set for 1000ms.